magento/pwa-studio
# [PWA-281] My Account: Wishlist (Skeleton)

**Should be merged after #2611, which includes left navigation auto close logic.**



## Description

As a shopper (authenticated), I want to view/modify/delete wishlists under my account. 

 

In-scope:

Create Skeleton for

Favorites View
Single
Multiple
Saved Item View
Requirements:

CE: only support for 1 List per shopper/account
EE: allows for Multiple List per shopper/account
Ensure additional stories are created to link new favorites/wishlist feature to cart and PDP workflows

Endstate:
When the shopper is authenticated and selects the "favorites list" menu item they are directed to the favorites view/page. 

## Related Issue





* [[PWA-281](https://jira.corp.magento.com/browse/PWA-281)] My Account: Wishlist (Skeleton)

## Acceptance 



### Verification Stakeholders

### Specification



### Verification Steps


In the scope of this PR, wishlist data is currently mocked and will always return a single list with no items in it
1. Login as a Customer
2. Using either of the menus, select Favorites Lists
3. Verify that skeleton is rendered with mocked data
4. On CE, verify there is no Create a list button

## Screenshots / Screen Captures (if appropriate)
![wishlist-mobile](https://user-images.githubusercontent.com/462953/90267184-d575bb00-de1a-11ea-86b4-d1022bbe3bbb.png)
![wishlist-desktop](https://user-images.githubusercontent.com/462953/90267188-d60e5180-de1a-11ea-8aab-ce948ed49578.png)
![wishlist-disabled](https://user-images.githubusercontent.com/462953/90267189-d6a6e800-de1a-11ea-9e5b-64e9679810f6.png)
![wishlist-fetchError](https://user-images.githubusercontent.com/462953/90267190-d6a6e800-de1a-11ea-94b1-efed646220fd.png)

## Checklist

* I have updated the documentation accordingly, if necessary.
